services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowPriorityRerouteSpec.groovy

```diff
@@ -56,7 +64,10 @@ class FlowPriorityRerouteSpec extends BaseSpecification {
         }
 
         and: "Reroute procedure was done based on the priority field"
-        flows.sort { it.priority }*.id == northbound.getAllFlows().sort { it.lastUpdated }*.id
+        int delay = protectedPath ? rerouteDelay * 2 : 1
```

Comment:
I absolutely don't understand what happens here

Reply:
I will use `WAIT_OFFSET` instead of the `delay` variable

Reply:
done